templates/module/helfi_tpr/tpr-unit--minimal.html.twig

```diff
@@ -14,8 +14,15 @@
     }]) %}
 {% endif %}
 
+{%
+  set card_tpr_classes = [
+  not unit_published ? 'card--unpublished',
```

Comment:
This variable could be more generic. Like `not content_entity_published ? 'card--unpublished',`

Reply:
Made it generic 🦖